0019-glossary/0019-glossary.md

```diff
@@ -0,0 +1,272 @@
+The application layer may provide guidance to sender and receiver in determining whether an exchange rate offered by a connector
+is reasonable or not.
+
+## Fulfillment
+See [conditional transfer](#conditional-transfer).
+
+## Destination (transfer, ledger, amount)
+The transfer/ledger/amount directly adjacent to the receiver.
+
+## Dispute
+Some ledgers, when administered by more than one authority, allow dispute over the ledger's state to occur naturally. This can be
+useful when the cost of resolving every small difference in accounting is much higher than the value of the differences.
+A transfer whose outcome is disputed among the ledger's administering authorities, can be resolved as 'disputed', in which case
+the sender's balance is decreased, and instead of increasing the receiver's balance, the sender's "disputed balance" is increased.
+Disputed balance can disappear over time when disputed transfers happen in the other direction, or when settled by out-of-band
+communication between the authorities who administer the ledger.
+Note that although individual ledgers can support disputed transfers, an interledger transaction as a whole can never be disputed;
+its final state can only be either 'executed' or 'canceled'.
+
+## Distributed ledger
+A ledger which is operated by a group of entities.
+
+## Executed
+One of the final states an interledger transactions can end up in.
+
+## Final state
+The state an interledger transaction ends up in. Has to be either
+[canceled](#canceled) or [executed](#executed).
+
+## Forward action
+For a sender, the forward action consists in creating an outgoing transfer.
+When a connector is notified that an incoming transfer has reached the 'prepared' state, its forward action is to propose a transfer on the next ledger,
+with the same hashlock condition, and the same ILP packet attached.
+For a receiver, there is no forward action.
+
+## Hop
+One connector-facilitated connection between one ledger and the next.
+The number of hops is always one less than the number of transfers, so a one-transfer ("local") transaction is a zero-hop transaction.
+
+## IL-RFC
+Interledger request for comments. A document describing a part of the protocol stack, or a topic related to it, such as this glossary. The official list of IL-RFCs is maintained in
+[gh:interledger/rfcs](https://github.com/interledgers/rfcs).
+
+## Incoming (transfer, ledger, amount)
+Relative to a connector, the transfer/ledger/amount that is located inbetween the connector and the sender, directly adjacent to the connector.
+For a receiver, the adjacent transfer/ledger/amount is always incoming.
+A sender does not have an incoming transfer/ledger/amount.
+
+## Interledger (in "Let's use Interledger for that!")
+The [Interledger protocol stack](#interledger-protocol-stack).
+
+## interledger (in "An interledger")
+Any network of two or more ledgers connected via the Interledger protocol stack. See also [Interledger (The)](#interledger-the)
```

Comment:
I think this is more the adjective when you're describing transactions that go across multiple ledgers